hugo-modules/core/utils/seo/private/get-tags.html

```diff
@@ -97,7 +97,7 @@
 {{/* Dealing with private setting */}}
 {{ with $seo }}
   {{ $value := "noindex, nofollow" }}
-  {{ $production := eq (getenv "HUGO_ENV") "production" }}
+  {{ $production := eq (hugo.Environment) "production" }}
```

Comment:
i think currently `hugo.Environment` only differentiates between development & production. This would lead to `index` / `follow` on preview & staging environments. 
Either we need to introduce more hugo environments or we should keep it as it is.

Reply:
We have used `HUGO_ENV` here and use `HUGO_ENVIRONMENT` in baseof.html and scripts.html. Which one is correct?

Reply:
hmm... i guess that `HUGO_ENVIRONMENT` is the same as `hugo.Environment` whereas `HUGO_ENV` is a custom name taken from [Huge](https://github.com/theNewDynamic/hugo-module-tnd-seo/blob/main/partials/private/GetTags.html#L103) and is kind of independent. But i get that the name is a bit confusing :)
Maybe we should stick to `hugo.Environment` or `HUGO_ENVIRONMENT` and introduce hugo environments for staging & develop?

Reply:
When HUGO_ENV is set to production, the noindex/nofollow values will be set based on the settings in contentful (old SEO model)
When it is not set to production, it will always use noindex/nofollow regardless of anything configured in contentful. Maybe the docs are a bit misleading ;)

Reply:
Ok I get it. When I read it a second time, the sentence makes sense to me. ;-) I now use `HUGO_ENV` here and have renamed all other places where we still used `HUGO_ENVIRONMENT`.